core/src/autogluon/core/metrics/__init__.py

```diff
@@ -386,7 +386,7 @@ def customized_log_loss(y_true, y_pred, eps=1e-15):
     if y_pred.ndim == 1:
         # First clip the y_pred which is also used in sklearn
         y_pred = np.clip(y_pred, eps, 1 - eps)
-        return (y_true * np.log(y_pred) + (1 - y_true) * np.log(1 - y_pred)).mean()
+        return - (y_true * np.log(y_pred) + (1 - y_true) * np.log(1 - y_pred)).mean()
```

Comment:
@Innixma The root cause of the error should be this line.

Reply:
Let me double check if it will fix the issue.

Reply:
Please test this on Tabular with the standard models, I suspect this is not the correct solution.

Reply:
Nevermind, this is indeed a bug.